docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,503 @@
+Scale: -100 to 0 needs improvements, 0 to 30 is good, 30 to 70 is great, and 70 to 100 is excellent." />
+
+<p class="blog-description">1563 out of 1589 answered</p>
+
+### What is the main benefit you get from using MUI?
+
+<img src="/static/blog/2021-survey/3.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="This question results in 2019 - Design: 248; Time: 193; DX: 128; Components: 89; Customizability: 53; Documentation: 25; Accessibility: 5; Typescript: 5; Community support: 4; This question results in 2021 - Time: 597; Design: 407; Components: 309; DX: 306; Customizability: 181; Documentation: 57; Community support: 26; Accessibility: 24; Performance: 17" />
+
+<p class="blog-description">1422 out of 1589 answered this question in 2021</p>
+
+Here's what you liked the most about the product in 2021:
+
+- **Community (x2.9):** While appreciation for the community grew by 290%, the community itself doubled in size over that same time frame.
+  It looks like we're witnessing a network effect: with twice as many users, we got three times more value from the community.
+- **Accessibility (x2.2):** The community has definitely noticed the work we have put in to make our components more accessible.
+  Special thanks to Sebastian Silbermann [(@eps1lon)](https://github.com/eps1lon) for his continued help!
+- **Customizability (x1.6):** Developers seem to be appreciating the new capabilities introduced in v5.
+  That said, we recognize that there is still more work to be done here, given the number of requests we've gotten to better solve this problem.
+- **Components (x1.6):** We'll keep adding new components to the library as long as you continue to appreciate them this much!
+  - Component quality (x1.0): You know that we set the bar very high for the quality of our components.
+    That can be challenge as we continue to grow, because we expect nothing but the best from our team.
+    Thankfully, this result shows that our new members have done an excellent job maintaining our standards.
+- **Time (x1.4):** This is an interesting one, because it's not an issue that we specifically addressed.
+  Could this just be a result of the growing pressure in the market to build faster? Or because we somehow made this value proposition more noticeable? It could also be related to the next item.
+- **Developer experience (x1.1):**
+  - Consistency (x1.75): We didn’t work on this dimension, and yet it grew.
+    It's reasonable to assume that because we have more components than ever before, developers have started to notice how much more consistent their work is when they use MUI.
+  - Ease of use (x1.0): This remains unchanged.
+    Could it be because ease of use is mostly defined by the React API (hooks), or is it because we haven't made any major API changes to optimize for ease of use? There could be a lot of work here for our new developer experience team.
```

Comment:
Developer relations is a subdiscipline of Developer Experience from my perspective.

Reply:
@mbrookes I like Developer Experience as the umbrella term, drawing from Kurt Kemple's perspective: [Redefining Developer Experience: Why DX is the New DevRel](https://theworst.dev/developer-experience)

Reply:
I love this ressource. I think that it's a matter of how we want to organize the team, knowing that the team organization is a tradeoff that at the end of the day shapes what we build.

IMHO, the DX team should care a lot about shaping the product (experience) for the developers, it would include technical writing, learning, dev rel, marketing. For comparison, I think that the design team should focus on all users, e.g. end-users, with a focus toward using the UI and UX design disiplines to solve problems. The Engineering, DX, and Design departments all gravitates around the product.